src/services/upper-bar-manager/upper-bar-manager.tsx

```diff
@@ -76,16 +76,20 @@ export class UpperBarManager {
     return icons.sort((a, b) => iconsOrder[a.displayName] - iconsOrder[b.displayName]);
   }
 
+  public getMorePluginButton() {
+    const rightControls = this.displayedBarComponentRefs.Playback.current?.base as HTMLElement;
+    const buttons = rightControls.querySelectorAll('[tabindex="0"]');
+    return Array.from(buttons).find((button) => button.classList.contains('playkit-more-icon_fR')) as HTMLButtonElement;
```

Comment:
I don't like idea with `button.classList.contains('playkit-more-icon_fR'))`
more-icon.component.tsx has `this.moreButtonRef`, so we can send callback method from upper-bar-manager.tsx to more-icon.component.tsx and keep reference of more button in upper-bar-manager.tsx. Then instead of parsing DOM we can get the more button from ref

Reply:
we can't set this.moreButtonRef the from more-icon.component.tsx since we need it global and in another components we are using instances of upper-bar-manager.
so i did it a bit different by adding refrence to moreIcon component in display-bar-component then use the ref from the existing displayBarComponent reference.